src/platform/platform_linux.cpp

```diff
@@ -154,6 +155,59 @@ void update_bridges(std::map<std::string, mp::NetworkInterfaceInfo>& networks)
 }
 } // namespace
 
+std::pair<QString, QString> mp::platform::Platform::parse_os_release(const QStringList& os_data)
+{
+    const QString id_field = "NAME";
+    const QString version_field = "VERSION_ID";
+
+    QString distro_id = "unknown";
+    QString distro_rel = "unknown";
+
+    for (const QString& line : os_data)
+    {
+        QStringList split = line.split('=', Qt::KeepEmptyParts);
+        if (split.length() == 2 && split[1].length() > 2) // Check for at least 1 char between quotes.
+        {
+            QString val = split[1].mid(1, split[1].length() - 2); // Removing quotes from input string.
+            if (split[0] == id_field)
+                distro_id = val;
+            else if (split[0] == version_field)
+                distro_rel = val;
+        }
```

Comment:
Good review points, I have made the updates. I would have liked to use `std::move` instead of the pointers but that will not avoid unneeded string splits.

Reply:
I have created a lambda to strip quotes, the name is explanatory enough to remove the comment. Copy elision optimization should take care of unneeded copying. Tests passing.